libs/langchain_v1/langchain/agents/middleware/types.py

```diff
@@ -120,12 +119,6 @@ class AgentMiddleware(Generic[StateT, ContextT]):
     tools: list[BaseTool]
     """Additional tools registered by the middleware."""
 
-    before_model_jump_to: ClassVar[list[JumpTo]] = []
```

Comment:
Did you not need these for resolving some annotation? What happens if the middleware is defined not via decorators?